src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -139,5 +140,60 @@ class SubgraphBaseStaticTest : public ov::test::SubgraphBaseTest {
         ov::test::SubgraphBaseTest::run();
     }
 };
+
+inline std::string setToString(const std::unordered_set<std::string> s) {
+    if (s.empty())
+        return {};
```

Comment:
```suggestion
inline std::string setToString(const std::unordered_set<std::string>& s) {
    return std::string("{") + ov::util::join(s) + "}";
```
Will return `{}` for empty set, the default separator ", " in `ov::util::join` can be replaced to "," if required.

Move implementation to cpp file.

Reply:
Updated.